CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Added the ability to use the latest USS encoding when saving an exiting file.
 - Removed the Cancel button and replaced it with X button on all dialogue pop-ups
 - Added toggle bar to show and hide File Explorer.
+- Added feature to open a file in new broswer tab
```

Comment:
2.8 is the previous release' version of the Editor, could you change this to 2.9 and update the package.json/package-lock.json accordingly?